### DISCUSSION TITLE
Version 3.0.0, 2022-12-22

### DISCUSSION THREAD

**Discussion by MartinThoma** [py-pdf/pypdf#1512] [Announcements]
### BREAKING CHANGES
-  Deprecate features with PyPDF2==3.0.0 (#1489)
-  Refactor Fit / Zoom parameters (#1437)

### New Features (ENH)
-  Add Cloning  (#1371) by @pubpub-zz
-  Allow int for indirect_reference in PdfWriter.get_object (#1490)

### Documentation (DOC)
-  How to read PDFs from S3 (#1509)
-  Make MyST parse all links as simple hyperlinks (#1506) by @mbromet
-  Changed 'latest' for 'stable' generated docs (#1495) by @olsonperrensen
-  Adjust deprecation procedure (#1487)

### Maintenance (MAINT)
-  Use typing.IO for file streams (#1498) by @thehale

## New Contributors
* @olsonperrensen made their first contribution in https://github.com/py-pdf/PyPDF2/pull/1495
* @thehale made their first contribution in https://github.com/py-pdf/PyPDF2/pull/1498
* @mbromet made their first contribution in https://github.com/py-pdf/PyPDF2/pull/1506

**Full Changelog**: https://github.com/py-pdf/PyPDF2/compare/2.12.1...3.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/py-pdf/PyPDF2/releases/tag/3.0.0'>Version 3.0.0, 2022-12-22</a>.</em>

**Comment by MartinThoma** [comment]
https://github.com/py-pdf/pypdf/issues/1488 is the next step

**Comment by carecavoador** [comment]
Wow, big news! Thank you to all the community and contributors!
